chapters/5-processing.tex

```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
 so other researchers may use them for their analysis as well.
 
 \section{Looking ahead}
-Once you have corrected, cleaned, 
-and de-identified one or several data tables,
-and compiled the documentation to support it,
+This chapter introduced a process of exploring, understanding, and correcting
+the data that you obtained from the field or from partners.
+This process, called data cleaning and processing,
+creates your first research outputs using original data:
+an analysis-ready dataset.
+This dataset is a clear representation of reality (it is ``tidy''),
+it faithfully represents the measurements it was intended to collect,
+and it does not expose the identities of the people described by it.
+You have also now taken the time to fully understand the patterns and structures
+in your data, and annotated and labeled them for your use and use by others.
+Once you have completed these steps and
+and compiled the documentation to accompany your work,
 you have created the first research output of your project: an original dataset.
```

Comment:
this repeats the line above (about creating the first research output). i suggest removing all of this, unless we want to add a separate point about documentation that is necessary to make this a useful output for the research process going forward

Reply:
```suggestion
You have also now taken the time to fully understand the patterns and structures
in your data, and annotated and labeled them for your use and use by others.
```